Exercise 10

kogoe1 opened the pull request.

BenRamchandani reviewed (changes requested):
Hi Kofi,

What you've done so far looks good, but you've only changed the what's displayed to the user, anyone can still make and move to do items by manually making requests to the server, since you don't check their role in the server side code (Part 2 Step 2).

Let me know if that's not clear.

kogoe1 commented on pyproject.toml:
```diff
@@ -12,6 +12,9 @@ Flask-WTF = "^0.14.3"
 pymongo = {extras = ["srv"], version = "^3.12.0"}
+oauthlib = "^3.1.1"
+Flask-Login = "^0.5.0"
+Flask-User = "^1.0.2"
```
Thanks Ben.
Have removed Flask-User from pyproject.toml file. oauthlib and Flask-Login are needed.

kogoe1 commented:
I have implemented Part 2 Step 2 now (i.e. added role checks at each endpoint).

A side note - could you set my user account in Azure so I can add it to Authenticator app again on a new phone please?
